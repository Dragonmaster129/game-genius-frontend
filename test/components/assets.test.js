import React from "react";
import { render, screen, fireEvent } from "@testing-library/react";
import Assets from "../../src/components/assets";
import externalData from "../sampledata";

test("assets component is the same as what it was", () => {
  render(<Assets props={externalData.assets} />);
  let assets = screen.getByRole("heading", { name: "Assets" });
  expect(assets).toBeDefined();
});
