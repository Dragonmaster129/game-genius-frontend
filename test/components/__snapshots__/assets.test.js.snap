// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`assets component is the same as what it was 1`] = `
<div
  className="v"
>
  <h2>
    Assets
  </h2>
  <h3>
    Savings
  </h3>
  <div
    className="stock"
  >
    <div
      className="hz"
    >
      <h3>
        Stocks/Mutual's/CDs
      </h3>
      <h3>
        Number of Shares
      </h3>
      <h3>
        Cost/Share
      </h3>
    </div>
    <div
      className="hz asset-item"
    >
      <div>
        OK4U
      </div>
      <div>
        1000
      </div>
      <div>
        1
      </div>
    </div>
  </div>
  <div
    className="re"
  >
    <div
      className="hz heading"
    >
      <h3>
        Real Estate
      </h3>
      <h3>
        Down Pay
      </h3>
      <h3>
        Cost
      </h3>
    </div>
    <div
      className="hz asset-item"
    >
      <div>
        3/2 HOUSE
      </div>
      <div>
        5000
      </div>
      <div>
        55000
      </div>
    </div>
    <div
      className="hz asset-item"
    >
      <div>
        3/2 HOUSE
      </div>
      <div>
        7000
      </div>
      <div>
        55000
      </div>
    </div>
  </div>
  <div
    className="business"
  >
    <div
      className="hz"
    >
      <h3>
        Business
      </h3>
      <h3>
        Down Pay
      </h3>
      <h3>
        Cost
      </h3>
    </div>
    <div
      className="hz asset-item"
    >
      <div>
        Auto Parts
      </div>
      <div>
        100000
      </div>
      <div>
        100000
      </div>
    </div>
    <div
      className="hz asset-item"
    >
      <div>
        Auto Parts
      </div>
      <div>
        120000
      </div>
      <div>
        120000
      </div>
    </div>
  </div>
</div>
`;
